observation_portal/requestgroups/serializers.py

```diff
@@ -31,83 +32,104 @@
 logger = logging.getLogger(__name__)
 
 
+def cerberus_validation_error_to_str(validation_errors: dict) -> str:
```

Comment:
the problem is since error_str is being pieced together recursively, it isn't just a simple list. I.e. if you only have one field in error, it might be a list, but if you have several fields in error at different levels of the structure, it would not be